### PR TITLE
SDT-35: Update existing Warrant Request workflow

### PR DESCRIPTION
**Pull request by kiran-yenigala-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SDT-35

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/WarrantService.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    private CMCApi cmcApi;
+
+    @Autowired
+    public WarrantService(CMCApi cmcApi) {
+        this.cmcApi = cmcApi;
+    }
+
+    @Override
+    public WarrantResponse warrantRequest(String authorization,
+                                          String serviceAuthorization,
```
Comment (paulridout) [inline consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/WarrantService.java:22]: These two parameters aren't on the corresponding method on the other service classes (e.g. JudgementRequestService.requestJudgment()) - are they needed?

File: consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/WarrantRequest.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
+package uk.gov.moj.sdt.cmc.consumers.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.Address;
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.SotSignature;
```
Comment (paulridout) [inline consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/WarrantRequest.java:9]: As these classes are used by more than one request type, should they be moved to a common package?

File: handlers/src/main/java/uk/gov/moj/sdt/handlers/WsCreateBulkRequestHandler.java
Comment (paulridout) [inline handlers/src/main/java/uk/gov/moj/sdt/handlers/WsCreateBulkRequestHandler.java:1]: Move the declaration of the concurrencyMap field and it's associated comment above the constructor method.

File: consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/WarrantService.java (hunk 1 above)
Comment (kiran-yenigala-hmcts) [reply consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/impl/WarrantService.java:22]: Better to have them

File: consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/WarrantRequest.java (hunk 2 above)
Comment (kiran-yenigala-hmcts) [reply consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/WarrantRequest.java:9]: done

File: handlers/src/main/java/uk/gov/moj/sdt/handlers/WsCreateBulkRequestHandler.java
Comment (kiran-yenigala-hmcts) [reply handlers/src/main/java/uk/gov/moj/sdt/handlers/WsCreateBulkRequestHandler.java:1]: done

**Review by paulridout — APPROVED** [review]
Changes look OK.

**Pull request merged**